setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
what am i missing? it looks like the only place `django-formtools` and `django-modelutils` is specified is this file. how would those two libraries get installed locally by running pip?